src/components/widgets/AssocListPopup/AssocListPopup.tsx

```diff
@@ -44,7 +44,6 @@ export interface IAssocListOwnProps extends Omit<PopupProps, 'bcName' | 'childre
 }
 
 export interface IAssocListProps extends IAssocListOwnProps {
-    showed: boolean,
```

Comment:
This will break all client apps that use this component: any changes to public API only allowed in major release, so instead of removal it should be marked as optional and deprecated:
```ts
/**
 * @deprecated TODO: Remove in 2.0.0, now handled by Widget.tsx
*/
showed?: boolean
```